.github/workflows/backport.yml

```
@@ -11,7 +11,7 @@ jobs:
     name: Backport
     steps:
       - name: Backport Bot
-        uses: Gaurav0/backport@v1.0.24
+        uses: Gaurav0/backport@v1.0.25
```

```suggestion
        uses: Gaurav0/backport@v1.0.26
```
I can see that it's at 26 now and this is what is also used in QGIS code repo. Is the issue fixed?